package.json

```diff
@@ -314,6 +314,11 @@
           "type": "number",
           "default": 60,
           "description": "The time Visual Studio Code will wait for the OmniSharp server to start. Time is expressed in seconds."
+        },
+        "omnisharp.maxProjectResults": {
+          "type": "number",
+          "default": 100,
```

Comment:
we could, perhaps `250`? Thing is you probably want to cap it at a reasonable number cause when you open against i.e. `c:\` it would take ages to scan if the limit is too high